src/Page/Simulator.elm

```diff
@@ -160,6 +160,20 @@ init maybeQuery session =
     )
 
 
+toCustomCountryMixFormInputs : Inputs.CustomCountryMixes -> CustomCountryMixInputs
+toCustomCountryMixFormInputs customCountryMixes =
+    { fabric =
+        customCountryMixes.fabric
+            |> Maybe.map (Co2.inKgCo2e >> String.fromFloat)
+    , dyeing =
+        customCountryMixes.dyeing
+            |> Maybe.map (Co2.inKgCo2e >> String.fromFloat)
+    , making =
+        customCountryMixes.making
+            |> Maybe.map (Co2.inKgCo2e >> String.fromFloat)
+    }
```

Comment:
Nit: this could be slightly simplified:

```elm
toCustomCountryMixFormInputs : Inputs.CustomCountryMixes -> CustomCountryMixInputs
toCustomCountryMixFormInputs { fabric, dyeing, making } =
    let
        mapToCo2e =
            Maybe.map (Co2.inKgCo2e >> String.fromFloat)
    in
    { fabric = mapToCo2e fabric
    , dyeing = mapToCo2e dyeing
    , making = mapToCo2e making
    }
```